src/HylianShield/Validator/Url/Network/InstructionSet.php

```diff
@@ -0,0 +1,273 @@
+<?php
+/**
+ * An instruction set for network validators.
```

Comment:
Please add some extra information about the intention of the class. Makes it easier to understand for others.